database/src/main/resources/changesets/metadata_journal.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="metadata_journal">
+        <createTable tableName="METADATA_JOURNAL">
+            <column autoIncrement="true" name="METADATA_JOURNAL_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_METADATA_JOURNAL"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="METADATA_KEY" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="METADATA_CALL_FQN" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_CALL_INDEX" type="INT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_CALL_ATTEMPT" type="INT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_VALUE" type="TEXT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_TIMESTAMP" type="DATETIME">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB"/>
+        </modifySql>
+    </changeSet>
+
+    <changeSet id="metadata_workflow_index" author="mcovarr">
+        <createIndex indexName="METADATA_WORKFLOW_IDX"
+                     tableName="METADATA_JOURNAL" unique="false">
+            <column name="WORKFLOW_EXECUTION_UUID"/>
+        </createIndex>
+    </changeSet>
```

Comment:
missed an index for workflowId + key, assuming we still want that per the comments below